src/app/campaign-details/campaign-details.component.html

```diff
@@ -177,91 +192,106 @@
-
-          <mat-tab *ngIf="campaign.updates.length > 0" label="Updates">
-            <ul class="c-updates" *ngFor="let update of campaign.updates">
-              <li>
-                <biggive-formatted-text>
-                  <h4 class="b-semi-bold b-mb-0">{{ update.modifiedDate | date : 'longDate' }}</h4>
-                  <p class="c-updates__description">{{ update.content }}</p>
-                </biggive-formatted-text>
-              </li>
-            </ul>
+            @if (campaign.additionalImageUris.length > 0) {
+              <div class="c-additional">
+                @for (image of campaign.additionalImageUris; track image) {
+                  <img
+                    [src]="image.uri | optimisedImage:850 | async"
+                    class="c-additional__images"
+                    >
+                }
+              </div>
+            }
+
+            @if (campaign.quotes.length > 0) {
+              <ul class="c-quotes">
+                @for (quote of campaign.quotes; track quote) {
+                  <li>
+                    <biggive-quote
+                      space-below="4"
+                      default-text-colour="black"
+                      [quote]="quote.quote"
+                      [attribution]="quote.person"
+                      >
+                    </biggive-quote>
+                  </li>
+                }
+              </ul>
+            }
           </mat-tab>
 
-          <mat-tab *ngIf="campaign.aims.length > 0" label="Aims">
-            <biggive-heading
-              space-above=6
-              space-below=0
-              colour="primary"
-              html-element="h3"
-              size="3"
-              align="left"
-              text="Campaign Aims"
-            >
-            </biggive-heading>
-
-            <ul class="c-aims" *ngFor="let aim of campaign.aims">
-              <li class="b-rt-0">{{ aim }}</li>
-            </ul>
-          </mat-tab>
-
-          <mat-tab *ngIf="campaign.budgetDetails.length > 0 || campaign.alternativeFundUse" label="Where's the money going?">
-
-            <biggive-heading
-              space-above=6
-              space-below=0
-              colour="primary"
-              html-element="h3"
-              size="3"
-              align="left"
-              text="Campaign Budget"
-            >
-            </biggive-heading>
-
-            <ul class="c-budget" *ngFor="let detail of campaign.budgetDetails">
-              <li class="c-budget__item">
-                <span class="c-budget__value b-semi-bold">{{ detail.amount | currency : campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo }}</span>
-                &ndash; {{ detail.description }}
-              </li>
-            </ul>
-
-            <div class="c-alternative-funds" *ngIf="campaign.alternativeFundUse">
+          @if (campaign.updates.length > 0) {
+            <mat-tab label="Updates">
+              @for (update of campaign.updates; track update) {
```

Comment:
https://angular.io/guide/control_flow#track-for-calculating-difference-of-two-collections